FirebaseAdmin/FirebaseAdmin/Auth/FirebaseAuth.cs

```diff
@@ -263,6 +263,37 @@ public async Task<FirebaseToken> VerifyIdTokenAsync(string idToken)
                 .ConfigureAwait(false);
         }
 
+        /// <summary>
+        /// Gets a <see cref="UserRecord"/> object containig information about the user who's
+        /// user ID was specified in <paramref name="uid"/>.
+        /// </summary>
+        /// <param name="uid">The user ID for the user who's data is to be retrieved..</param>
+        /// <returns>A task that completes with a <see cref="UserRecord"/> representing
+        /// a user with the specified user ID.</returns>
+        /// <exception cref="ArgumentException">If user ID argument is null or empty.</exception>
+        /// <exception cref="FirebaseException">If a user cannot be found with the specified user ID.</exception>
+        public async Task<UserRecord> GetUserAsync(string uid)
+        {
+            return await this.userManager.Value.GetUserById(uid);
```

Comment:
You need to wrap this in `IfNotDeleted()`. See `SetCustomUserClaimsAsync()` method for an example.